# moto[server] 5.0.0
#
# breaks with
# Traceback (most recent call last):
#   File "/usr/local/bin/sbx-mocks3-serve", line 5, in <module>
#     from shoobx.mocks3.run import serve
#   File "/usr/local/lib/python3.11/site-packages/shoobx/mocks3/__init__.py", line 17, in <module>
#     from moto.backends import BACKENDS
# ImportError: cannot import name 'BACKENDS' from 'moto.backends' (/usr/local/lib/python3.11/site-packages/moto/backends.py)
#
# Moto v5 is incompatible with current mocks3 implementation
-e .[test]
annotated-types==0.6.0
attrs==23.2.0
aws-sam-translator==1.84.0
aws-xray-sdk==2.12.1
blinker==1.7.0
boto3==1.34.34
botocore==1.34.34
certifi==2024.2.2
cffi==1.16.0
cfn-lint==0.85.0
charset-normalizer==3.3.2
click==8.1.7
coverage==7.4.1
cryptography==42.0.2
docker==7.0.0
ecdsa==0.18.0
flask==3.0.2
flask-cors==4.0.0
freezegun==1.4.0
graphql-core==3.2.3
idna==3.6
iso8601==2.1.0
itsdangerous==2.1.2
jinja2==3.1.3
jmespath==1.0.1
jschema-to-python==1.2.3
jsondiff==2.0.0
jsonpatch==1.33
jsonpickle==3.0.2
jsonpointer==2.4
jsonschema==4.21.1
jsonschema-path==0.3.2
jsonschema-specifications==2023.12.1
junit-xml==1.9
junitxml==0.7
lazy-object-proxy==1.10.0
markupsafe==2.1.5
mock==5.1.0
moto[server]==4.2.10
mpmath==1.3.0
networkx==3.2.1
openapi-schema-validator==0.6.2
openapi-spec-validator==0.7.1
packaging==23.2
pathable==0.4.3
pbr==6.0.0
py-partiql-parser==0.4.2
pyasn1==0.5.1
pycparser==2.21
pydantic==2.6.0
pydantic-core==2.16.1
pyparsing==3.1.1
python-dateutil==2.8.2
python-jose[cryptography]==3.3.0
python-subunit==1.4.4
pytz==2023.4
pyyaml==6.0.1
referencing==0.31.1
regex==2023.12.25
requests==2.31.0
responses==0.24.1
rfc3339-validator==0.1.4
rpds-py==0.17.1
rsa==4.9
s3transfer==0.10.0
sarif-om==1.0.4
six==1.16.0
sshpubkeys==3.3.1
sympy==1.12
testtools==2.7.1
typing-extensions==4.9.0
urllib3==2.0.7
werkzeug==3.0.1
wrapt==1.16.0
xmltodict==0.13.0
zope-exceptions==5.0.1
zope-interface==6.1
zope-testrunner==6.2.1
